### PR TITLE
Update 1.0.0.0-beta1 release notes

### PR DESCRIPTION
**Pull request by hsiang9431-amzn**
…ta1.md

##  opensearch-project/security-dashboards-plugin pull request intake form
_Please provide as much details as possible to get feedback/acceptance on your PR quickly_

 
 1. __Category:__ (_Enhancement, New feature, Bug fix, Test fix, Refactoring, Maintenance, Documentation_)
Documentation
 
 
 2. __Github Issue # or road-map entry, if available:__
N/A

 3. __Description of changes:__
Update content for `1.0.0.0-beta1` release notes

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-security-dashboards-plugin.release-notes-1.0.0.0-beta1.md [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
-* Update Unit tests workflow to incorporate name and version change
-* Remove unused build scripts
-* Add new scripts for fixing artifact zip file name
+### Bug Fixes
+* Revert "Add opendistro/ism and rollup actions" [#714](https://github.com/opensearch-project/security-dashboards-plugin/pull/714)
+* Making sure SAML cookie uses saml session timeout value if it is available [#715](https://github.com/opensearch-project/security-dashboards-plugin/pull/715) [#717](https://github.com/opensearch-project/security-dashboards-plugin/pull/717)
+* Cherry-pick fix of saving tenant option from main to 7.9.1 branch [#716](https://github.com/opensearch-project/security-dashboards-plugin/pull/716)
+* Save tenant selection by default and remove the checkbox [#722](https://github.com/opensearch-project/security-dashboards-plugin/pull/722)
+* Fix github integration test for 1.13.0.1 [#723](https://github.com/opensearch-project/security-dashboards-plugin/pull/723)
+* Updated logic for splitting SAML JWT token. This should not throw the… [#730](https://github.com/opensearch-project/security-dashboards-plugin/pull/730) [#731](https://github.com/opensearch-project/security-dashboards-plugin/pull/731)
```
Comment (cliu123) [inline release-notes/opensearch-security-dashboards-plugin.release-notes-1.0.0.0-beta1.md:15]: Please split them to 2 rows.

Comment (hsiang9431-amzn) [reply release-notes/opensearch-security-dashboards-plugin.release-notes-1.0.0.0-beta1.md]: fixed

File: release-notes/opensearch-security-dashboards-plugin.release-notes-1.0.0.0-beta1.md [hunk 2]
```diff
@@ -1,18 +1,25 @@
 ## Version 1.0.0.0-beta1
 
 ### Enhancements
-* Migrates plugin name from `Opendistro` to `Open Search`
```
Comment (cliu123) [inline release-notes/opensearch-security-dashboards-plugin.release-notes-1.0.0.0-beta1.md:4]: Why are we removing items? Were they added unexpectedly?

Comment (hsiang9431-amzn) [reply release-notes/opensearch-security-dashboards-plugin.release-notes-1.0.0.0-beta1.md:4]: there is a PR for that: name change to opensearch dashboard #743

**Review by cliu123 — APPROVED** [review]

**Pull request merged**